.travis.yml

```diff
@@ -7,5 +7,6 @@ go:
   - master
 env:
   - TAGS=""
+  - TAGS="-tags appengine"
```

Comment:
I don't think we really care about special appengine support these days. I filed #46.